introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my username is KevinDyer. I need to make this a longer message so I am...derp!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.